client/lib/gsuite/get-google-expiry-date.ts

```diff
@@ -0,0 +1,5 @@
+import type { ResponseDomain } from 'calypso/lib/domains/types';
+
+export function getGoogleExpiryDate( domain: ResponseDomain ): string | null {
```

Comment:
It may be worth accepting `undefined` and `SiteDomain` as well. This applies to all of the functions, so I won't duplicate the comment.
```suggestion
export function getGoogleExpiryDate( domain: ResponseDomain | SiteDomain | undefined ): string | null {
```